replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorHistoryImpl.java

```diff
@@ -91,6 +99,24 @@ public void addReplicationEvent(ReplicationStatus replicationStatus) {
     }
   }
 
+  @Override
+  public void removeReplicationEvent(String replicatorid) {
+    try {
+      provider.delete(new DeleteRequestImpl(replicatorid));
+    } catch (IngestException e) {
+      LOGGER.debug("Error deleting replication history item {}", replicatorid, e);
```

Comment:
FYI, you could have left the log at error and simply added another log at debug with the exception in order to get the stack trace. My comment was just to not log the exception at anything but debug.